src/Microsoft.AspNetCore.Http.Features/IHttpRequestFeature.cs

```diff
@@ -49,6 +49,11 @@ public interface IHttpRequestFeature
         string QueryString { get; set; }
 
         /// <summary>
+        /// The request target as it was sent in the HTTP request.
+        /// </summary>
+        string RawTarget { get; set; }
```

Comment:
That's ok in this case, it's not like we want people to use it. It's only the emergency backup.

That said, I'm sure @blowdart has some very strong language to put in the comments to the effect of `Don't even think about using this, you won't do it right and then you'll get hacked.`